dotCMS/src/main/java/com/dotcms/content/business/ContentletJsonAPIImpl.java

```diff
@@ -183,11 +190,30 @@ ImmutableContentlet toImmutable(
             } else {
                 Logger.debug(ContentletJsonAPIImpl.class,
                         String.format("Unable to set field `%s` as it wasn't set on the source contentlet", field.name()));
+
+                if(Config.getBooleanProperty(JSON_NUMERIC_FIELD_DEFAULT_TO_ZERO, true)) {
```

Comment:
if this is being called a lot, it would be better to encapsulate on a constant o lazy.
In general would be ok to restart to get this kind of config